test/app.all.js

```diff
@@ -1,10 +1,10 @@
 
-var express = require('../')
-  , request = require('supertest');
+const express = require('../')
+const request = require('supertest');
 
 describe('app.all()', function(){
   it('should add a router per method', function(done){
-    var app = express();
+    const app = express();
```

Comment:
I was wondering is there any particular reason why we can use a beforeEach() hook and put the const app = express();

Reply:
that would make to much sense. I'll refactor that. thanks

Reply:
I just looked at this again. I'm not sure it makes sense to do a before each just for one of code. I'd have to create an uninitialized var outside of the `it` block and then initialize app inside the beforeEach.  Seems like that would just add unnecessary lines to file.

Reply:
Thanks very much for replying. I understand what you mean, I realized that you would also have to change the app to a var instead of a const if you did what i was suggesting. I was thinking of code reuse, but perhaps in this case its one line so its fine. It was my first time commenting on an open source project, so i really do appreciate your comment.

Reply:
> change the app to a var instead of a const

actually, it could be a `let`. It's like a `var` but it's blocked scoped instead of function scoped (in line with how `const` works) :)

Reply:
Yeah, although I would personally recommend staying functional and try to make everything `const` and immutable 👍

Reply:
awesome, thanks for the explanation.